### PR TITLE
Minor cleanup in frozen collection code

### PR DESCRIPTION
**Pull request by geeknoid**

### CODE REVIEW

File: src/libraries/System.Collections.Immutable/src/System/Collections/Frozen/KeysAndValuesFrozenDictionary.cs [hunk 1]
```diff
@@ -2,13 +2,12 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 
 using System.Collections.Generic;
-using System.Collections.Immutable;
 using System.Diagnostics;
 
 namespace System.Collections.Frozen
 {
     /// <summary>Provides a base class for frozen dictionaries that store their keys and values in dedicated arrays.</summary>
-    public abstract class KeysAndValuesFrozenDictionary<TKey, TValue> : FrozenDictionary<TKey, TValue>, IDictionary<TKey, TValue>
```
Comment (stephentoub) [inline src/libraries/System.Collections.Immutable/src/System/Collections/Frozen/KeysAndValuesFrozenDictionary.cs:11]: I'm surprised this being public wasn't caught by our ref contract validation. @ViktorHofer, do you know why that wouldn't have shown up?

File: src/libraries/System.Collections.Immutable/src/System/Collections/Frozen/DefaultFrozenSet.cs [hunk 2]
```diff
@@ -9,8 +9,8 @@ namespace System.Collections.Frozen
     /// <typeparam name="T">The type of the values in the set.</typeparam>
     internal sealed class DefaultFrozenSet<T> : ItemsFrozenSet<T, DefaultFrozenSet<T>.GSW>
     {
-        internal DefaultFrozenSet(HashSet<T> source, IEqualityComparer<T> comparer) :
-            base(source, comparer)
+        internal DefaultFrozenSet(HashSet<T> source, IEqualityComparer<T> comparer)
+            : base(source, comparer)
```
Comment (stephentoub) [inline src/libraries/System.Collections.Immutable/src/System/Collections/Frozen/DefaultFrozenSet.cs:13]: This isn't typically a style we use elsewhere.

File: src/libraries/System.Collections.Immutable/src/System/Collections/Frozen/KeysAndValuesFrozenDictionary.cs (hunk 1 above)
Comment (ViktorHofer) [reply src/libraries/System.Collections.Immutable/src/System/Collections/Frozen/KeysAndValuesFrozenDictionary.cs:11]: Looking...

**Review by stephentoub — APPROVED** [review]

File: src/libraries/System.Collections.Immutable/src/System/Collections/Frozen/KeysAndValuesFrozenDictionary.cs (hunk 1 above)
Comment (stephentoub) [reply src/libraries/System.Collections.Immutable/src/System/Collections/Frozen/KeysAndValuesFrozenDictionary.cs:11]: Spoke offline with Viktor... he's working on a fix.

**Pull request merged**